asylum-connect/control-panel
# Feature/1193239233281585/autocomplete

## Description

Adds typeahead feature to name filter input

- Asana ticket: https://app.asana.com/0/1132189118126148/1193239233281585/f

## PR Checklist




## How to Test

1. Navigate to the AC Control Panel
2. Log in
3. Start typing in the "Name" input in the "Filter Organizations" section.
4. Observe that suggestions are being presented as you type.
5. a. Select an organization from the list.
   b. Click "Search"
   c. Observe that the results are relevant based on search query
